src/main/java/com/example/restapi/controller/EmployeeController.java

```diff
@@ -0,0 +1,60 @@
+package com.example.restapi.controller;
+
+import com.example.restapi.entities.Employee;
+import com.example.restapi.services.EmployeeService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/api/v1")
+public class EmployeeController {
+
+	@Autowired
+	private EmployeeService employeeService;
+
+	@GetMapping("/employee")
+	public List<Employee> retrieveAllEmployees() {
```

Comment:
it is rarely allowed to get all data from a resource without `Paging` or Filterring, imaging we have 10_000 employees, returning all of them to the response is resources waste and slow.

Let's try something new with `Paging` to get all employees.

Reply:
Fixed